animation_nodes/nodes/gpencil/gp_strokes_from_splines.py

```diff
@@ -0,0 +1,35 @@
+import bpy
+from ... base_types import AnimationNode, VectorizedSocket
+from ... data_structures import GPStroke, VirtualDoubleList, DoubleList
+
+class GPStrokesFromSplinesNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GPStrokesFromSplinesNode"
+    bl_label = "GP Strokes From Splines"
+    codeEffects = [VectorizedSocket.CodeEffect]
+
+    useSplineList: VectorizedSocket.newProperty()
+
+    def create(self):
+        socket = self.newInput(VectorizedSocket("Spline", "useSplineList",
+            ("Spline", "spline"), ("Splines", "splines")))
+        socket.defaultDrawType = "PROPERTY_ONLY"
+        socket.dataIsModified = True
+
+        self.newOutput(VectorizedSocket("GPStroke", "useSplineList",
+            ("Stroke", "stroke"),
+            ("Strokes", "strokes")))
+
+    def getExecutionCode(self, required):
+        return "stroke = self.strokesFromSplines(spline)"
+
+    def strokesFromSplines(self, spline):
+        if spline is None: return GPStroke()
+
+        vertices = spline.points
+        amount = len(vertices)
+        strengths = VirtualDoubleList.create(1, 1).materialize(amount)
+        pressures = DoubleList.fromValues(spline.radii)
+        uvRotations = VirtualDoubleList.create(0, 0).materialize(amount)
```

Comment:
No need to use virtual lists here. For `strengths` and `uvRotations`, just fill a list. For `pressures`, just copy `spline.radii`.

Reply:
Omar, spline.radii is a _FloatList_ but pressures is _DoubleList_, so _copy_ gives an error. Is anyway to convert them?

Reply:
I didn't realize this was the case. Can you rewrite the GP structures to use Float Lists?

Reply:
Yes, I'll make the changes.

Reply:
Omar, it does not work. For example, Float Range gives the _Double List_, so we can't it to set _strengths_ which is _FloatList_,
![Screenshot from 2020-02-22 20-00-34](https://user-images.githubusercontent.com/30294746/75094102-0385eb00-55ae-11ea-9396-8b57c672c942.png)

Maybe that is the reason, the conversion is used in _Set Spline Radius_,
https://github.com/JacquesLucke/animation_nodes/blob/8c25c75f266e02b46350ac3fb799da6747976b85/animation_nodes/nodes/spline/set_spline_radius.py#L56-L58

Reply:
Ok, do the conversion as needed.

Reply:
Done.